src/target/s32k3xx.c

```diff
@@ -0,0 +1,271 @@
+/*
+ * This file is part of the Black Magic Debug project.
+ *
+ * Copyright (C) 2015  Black Sphere Technologies Ltd.
+ * Written by Gareth McMullin <gareth@blacksphere.co.nz>
```

Comment:
These should have their attribution updated to have the right copyright year, you as the author, and 1b² as the copyright holder.